spec/unit/pops/types/type_calculator_spec.rb

```diff
@@ -578,7 +585,7 @@ class Foo
       end
 
       it 'Any is not assignable to anything but Any' do
```

Comment:
seems to test not assignable to anything but Any and Optional (implied Optional[Any]) - change the title of the test?